modules/ssh/ssh.go

```diff
@@ -217,6 +326,65 @@ func FetchContentsOfFileE(t *testing.T, host Host, useSudo bool, filePath string
 	return CheckSshCommandE(t, host, command)
 }
 
+func listFileInRemoteDir(t *testing.T, sshSession *SshSession, options ScpDownloadOptions) ([]string, error) {
+	logger.Logf(t, "Running command %s on %s@%s", sshSession.Options.Command, sshSession.Options.Username, sshSession.Options.Address)
+
+	var result []string
+
+	if err := setUpSSHClient(sshSession); err != nil {
+		return result, err
+	}
+
+	if err := setUpSSHSession(sshSession); err != nil {
```

Comment:
Is there a reason you're running all these steps manually instead of [CheckSshCommand](https://github.com/gruntwork-io/terratest/blob/master/modules/ssh/ssh.go#L95)?

Reply:
Nope, no real reason when getting the list of remote files. Just did the same thing I was doing when getting binary because I was on auto-pilot :-)

Switched to using CheckSshCommand.